app/controllers/project/messages/message.js

```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
+import { alias } from '@ember/object/computed';
+
+export default Controller.extend({
+  conversation: alias('model')
```

Comment:
ember inspector expects a `model` property on the controller, so I started using this alias instead of `setupController` we usually do. If we don't want this, I'm fine with reverting to `setupController`